pkg/injection/startup/secret.go

```diff
@@ -29,6 +29,8 @@ type SecretConfig struct {
 	HostGroup           string            `json:"hostGroup"`
 	InitialConnectRetry int               `json:"initialConnectRetry"`
 	EnforcementMode     bool              `json:"enforcementMode"`
+	CSIMode             bool              `json:"CSIMode"`
```

Comment:
```suggestion
	CSIMode             bool              `json:"csiMode"`
```

Reply:
apart from this suggestion it also LGTM.

Reply:
good catch